packages/integration_test/test/binding_test.dart

```diff
@@ -100,6 +100,13 @@ Future<void> main() async {
         expect(integrationBinding.defaultTestTimeout, newTimeout);
       });
     });
+
+    // TODO(jiahaog): Remove when https://github.com/flutter/flutter/issues/66006 is fixed.
+    testWidgets('root widgets are wrapped with a RepaintBoundary', (WidgetTester tester) async {
+      await tester.pumpWidget(const Placeholder());
+
+      expect(find.byType(RepaintBoundary), findsOneWidget);
+    });
```

Comment:
Rather than this test, can we just add a golden test for this?

Just pump a simple/hello world app, use the screenshot method, and use the Skia gold comparison logic to compare it.

Reply:
Sure, I can do that. Wouldn't it be clearer that we're specifically asserting for the `RepaintBoundary` by doing it in code here though?

Reply:
Sure, but I think the idea is that we care if the screen shot works, and we don't particularly care where a repaint boundary is :)

Reply:
That makes sense. I couldn't quite figure out how to get `matchesGoldenFile` to work with Skia gold though (following https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package%3Aflutter). cc @Piinks - is the Skia golden integration supported for non-`package:flutter` packages like this? If not I can just check in the png into the `tests/` directory alongside this test, but not sure if that's the best practice for this repo.

Reply:
The Skia Gold integration does not currently work outside of the flutter package, no. It is set up with the flutter_test_config.dart in flutter/test. Maybe we can expand it? Not sure if there will be more integration tests requiring Skia support.